### PR TITLE
Fix SorterTest

### PR DESCRIPTION
**Pull request by marcushoepfner**
Fix TestLabelProviderTooltips which might influence the SorterTest. TestLabelProviderTooltips caused an NPE because it did not have a backGroundColor. This NPE could just be seen in the console.

Fix TestLabelProvider which did not dispose the boldFont correctly. Also could be seen in the console.

Issue: https://github.com/eclipse-platform/eclipse.platform.ui/issues/809

### CODE REVIEW

**Comment by BeckerWdf** [comment]
"Today is a freeze day". ;-)

**Review by BeckerWdf — APPROVED** [review]

**Comment by BeckerWdf** [comment]
Why do we have this

image

twice?

**Comment by marcushoepfner** [comment]
Can I merge this? One test on windows is failing (seems not to be related).

File: tests/org.eclipse.ui.tests.navigator/src/org/eclipse/ui/tests/navigator/extension/TestLabelProvider.java [hunk 1]
```diff
@@ -166,9 +166,13 @@ public Font getFont(Object element) {
...
 	@Override
 	public void dispose() {
-		if (_throw)
-			throw new RuntimeException("Throwing...");
+		if (_runnable != null)
+			_runnable.run();
+		boldFont.dispose();
 		boldFont = null;
+
+//		font.dispose();
```
Comment (jukzi) [inline tests/org.eclipse.ui.tests.navigator/src/org/eclipse/ui/tests/navigator/extension/TestLabelProvider.java:174]: delete commented code?

**Comment by jukzi** [comment]
> Can I merge this? One test on windows is failing (seems not to be related).

When you see a test fails please make sure that an Issue for that exist.

**Comment by marcushoepfner** [comment]
> > Can I merge this? One test on windows is failing (seems not to be related).
> 
> When you see a test fails please make sure that an Issue for that exist.

It is always other failing tests:
- https://github.com/eclipse-platform/eclipse.platform.ui/actions/runs/5653230011/job/15315312687
- https://github.com/eclipse-platform/eclipse.platform.ui/actions/runs/5587249759/job/15133265583
Should I really create an issue for them?

**Comment by jukzi** [comment]
> * Should I really create an issue for them?

yes please

**Comment by marcushoepfner** [comment]
> > * Should I really create an issue for them?
> 
> yes please

Done: https://github.com/eclipse-platform/eclipse.platform.ui/issues/962

**Pull request merged**